NEWS

```diff
@@ -10,6 +10,7 @@ Current git version
   * detection of external panels
   * new command: apply_rules
   * new command: export (convenience wrapper around setenv)
+  * new client attribute: key_inactive (negation of keymask)
```

Comment:
You're missing an "s" here ;-)

Reply:
Thanks! I've fixed it directly in `master`.